unittests/snapshot_tests.cpp

```diff
@@ -472,7 +472,8 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_compatible_versions, SNAPSHOT_SUITE, snapshot
       auto old_snapshot = SNAPSHOT_SUITE::load_from_file("snap_" + version);
       BOOST_TEST_CHECKPOINT("loading snapshot: " << version);
       snapshotted_tester old_snapshot_tester(chain.get_config(), SNAPSHOT_SUITE::get_reader(old_snapshot), ordinal++);
-      verify_integrity_hash<SNAPSHOT_SUITE>(*chain.control, *old_snapshot_tester.control);
+      BOOST_REQUIRE_EQUAL(old_snapshot_tester.control->head_block_num(), chain.control->head_block_num());
```

Comment:
This is problematic and suggests that instead of addressing a root problem in our testing we decided to remove most of the purpose of the test. 

The integrity check would exhaustively check all state covered by snapshots between the two instances.  If it was failing that does positively indicate that there is a breakage in the _portable_ nature of these snapshots. 

We know that, in this case, it was drift in the construction of the snapshots and *probably* not in the snapshot system itself but by removing this check in favor for a far less complete test we cannot know that for sure.

At a minimum, we need to concretely verify that there is no drift in snapshot construction and create a good regression test for it. 

I am okay with removing and rethinking this test, I am not in support of just removing 99% of the important checks.